LICENSE

```diff
@@ -0,0 +1,7 @@
+Copyright (c) 2018 David Pedersen
```

Comment:
I know next to nothing about licenses, but shouldn’t this say 2019?

Reply:
Yes, I think 2019 is correct. I initially thought the date would be the date of first release on crates.io, however https://choosealicense.com/licenses/mit/ suggests using the current year when applying the copyright. Also makes sense to use the date of the first release where the copyright is included, thinking again. Not an expert myself - but I believe 2019 is the right way to go. Thank you for pointing it out! Updated the PR.